src/components/camera/camera.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {
+  Container,
+  Info,
+  Title,
+  SubTitle,
+  ScanTitle,
+  CameraContainer,
+  Input,
+  Button,
+} from "./camera.styles";
+
+/**
+ * Camera Component
+ * @author [Colburn Sanders](https://github.com/colburncodes)
+ */
+
+export const Camera = ({ title, caption, subTitle, inputText, buttonText }) => {
```

Comment:
Is this still a work in progress and we need to implement the webcam library, or are you intending to do that in a separate PR? My preference would be to keep this PR open as a `draft` so that I can comment on it as you work, but wait until component is working to merge it